app/System/Report/Providers/HtmlProvider.php

```diff
@@ -0,0 +1,11 @@
+<?php
+
+namespace App\System\Report\Providers;
+
+class HtmlProvider implements FormatProviderInterface
+{
+    public function formResponse(array $weatherReportArray)
+    {
+        return view('Report\report', ['report' => $weatherReportArray]);
```

Comment:
In big applications I would restrict helpers usages, and receive view builder from the container